pointcloud_sensor:
  base_setup:
    tf_name: "base_sensorring"
    print_topology: true
    enforce_topology: false
    frequency_tof_hz: 0.0 # max frequency ~ 13.6...14.1 Hz (0.0 -> free running)
    frequency_thermal_hz: 5.0 # max frequency ~ 5.0 Hz
    timeout_ms: 1000
    enable_brs: true
  thermal_config:
    auto_min_max: true
    use_eeprom_file: false
    use_calibration_file: false
    eeprom_file_dir: ".eduart/sensorring/eeprom"
    calibration_file_dir: ".eduart/sensorring/calibration"
    scale_t_min_deg: 20.0
    scale_t_max_deg: 35.0
  led_config:
    initial_mode: 9 # Off = 0, Dimmed = 1, HighBeam = 2, FlashAll = 3, FlashLeft = 4, FlashRight = 5, Pulsation = 6, Rotation = 7, Running = 8, MapDistance = 9
    initial_color: [64, 64, 64]
  topology:
    nr_of_interfaces: 1
    can_interfaces:
      can_interface_0:
        # interface_type: "usbtingo"
        #interface_name: "0x1731a1f1"
        interface_type: "socketcan"
        interface_name: "can0"
        orientation: "right"
        nr_of_sensors: 2
        sensors:
          sensor_0:
            enable_tof: true
            enable_thermal: false # true
            enable_light : true
            rotation: [0.0, 0.7, -90.0]
            translation: [0.1, 0.0, 0.0]
          sensor_1:
            enable_tof: true
            enable_thermal: false
            enable_light : true
            rotation: [0.0, 0.0, -90.0]
            translation: [0.177, 0.0, 0.0]
          sensor_2:
            enable_tof: true
            enable_thermal: false
            enable_light : true
            rotation: [0.0, 0.0, -90.0]
            translation: [0.254, 0.0, 0.0]